azext_iot/tests/test_iot_central_int.py

```diff
@@ -220,6 +220,20 @@ def test_central_device_methods_CRD(self):
 
         self._delete_device(device_id)
 
+    def test_central_user_methods_CRD(self):
+        (user_id, email) = self._create_user()
+
+        self.cmd(
+            "iot central app user show --app-id {} --id {}".format(APP_ID, user_id),
```

Comment:
What about a test for list users? To at least see if the created user shows up on the list.